modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -16,33 +16,42 @@ def create_a_record(fqdn, ip)
       if found = dns_find(fqdn)
         raise(Proxy::Dns::Collision, "#{fqdn} is already used by #{found}") if found != ip
       else
-        zone = match_zone(fqdn)
+        zone = match_zone(fqdn, enum_zones)
         msg = "Added DNS entry #{fqdn} => #{ip}"
         cmd = "/RecordAdd #{zone} #{fqdn}. A #{ip}"
         execute(cmd, msg)
+        nil
       end
     end
 
-    # noop
-    def create_ptr_record(fqdn, ip)
-      found = dns_find(ip)
-      raise(Proxy::Dns::Collision, "#{ip} is already used by #{found}") if found && found != fqdn
-      true
+    def create_ptr_record(fqdn, ptr)
+      ip = ptr_to_ip ptr
+      if found = dns_find(ip)
```

Comment:
> fix issues with multiple mappings

How should I best proceed with his one in case if A records?
 - return first / last mapping to #create
 - fail altogether

I rather fail in this case...